docs/partner-apps/bundleb2b/b2b-edition.mdx

```diff
@@ -1,54 +1,62 @@
+This section gives details on how to edit the DOM, CSS, and display text in a B2B Edition Stencil theme, and how to define JavaScript lifecycle methods. It assumes you are familiar with editing Stencil themes locally and pushing them to a sandbox or production store using the Stencil CLI.
 
-### Prerequisites for customizing B3
+### Prerequisites for customizing B2B Edition
 
 - [Node.js 14+ and npm](https://nodejs.org/en/download/releases/)
 - The [Stencil CLI](/stencil-docs/installing-stencil-cli/installing-stencil)
-- A [Stencil API token](https://support.bigcommerce.com/s/article/Store-API-Accounts#creating), to push your customized B3-optimized theme to a store 
-- A text editor
-- Access to your BigCommerce store control panel with store owner permissions
+- A [Stencil API token](>>>>) 
+- A text editor or IDE, such as [VS Code](https://visualstudio.microsoft.com)
+- Access to a BigCommerce store control panel with store owner permissions
 
 ### Customizing page containers
 
-B3 renders client-facing pages and elements by mounting fixed containers to BigCommerce themes using JavaScript. You can change the placement of select B3 elements by specifying the DOM node on which you want B3 to mount the container using the following steps:
+B2B Edition for Stencil renders client-facing pages and elements by mounting fixed containers to BigCommerce themes using JavaScript. B2B Edition elements and containers are _B3 objects_. You can change the placement of select B3 elements by specifying the DOM node on which you want B2B Edition to mount the container using the following steps:
 
 1. Insert `window.b3themeConfig.useContainers = {}` into your theme's `assets/js/global.js` file. 
 2. Within `window.b3themeConfig.useContainers = {}`, create a property with the name of the B3 container as its key and the selector for the theme element on which it will mount as its value. 
 
 When done, the object will resemble the following:
 
-```jsx filename="Example: Specify a custom mount node for the dashboard container" showLineNumbers copy
+```js filename="Example: Specify a custom mount node for the dashboard container" showLineNumbers copy
 window.b3themeConfig.useContainers = {
  /* B3 will append the dashboard container to the first returned DOM node with a class of "page" that is a descendant of an element with the class of "container" */
 	'dashboard.container': '.container .page',
 }
 ```
 
-For a list of B3 container names and their default placements, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/containers.html).
+For a list of B3 container names and their default placements, see the [B3 containers reference](https://developer.bundleb2b.net/storefront/containers.html).
```

Comment:
This is tricky because the objects are still called `b3`. I tried to differentiate where we're talking about the code objects vs B2B Edition as a product.

Same comment for all of these.

Reply:
Gotcha!

Would a glossary section in the top of the page be an idea?
ie.

In this document we will use B2B Edition to describe the 'product' and B3 to describe objects available via the B2B Edition API.

Reply:
that is an idea.

Reply:
see line 53.